Base/AutoUpdateChecker.cs

```diff
@@ -15,10 +15,18 @@ public static void CheckForUpdate(bool force = false, bool silent = false)
         {
             String hash = (Environment.UserName + Environment.MachineName).GetHashCode().ToString();
             if (Properties.Settings.Default.CacheId == "0") Properties.Settings.Default.CacheId = Guid.NewGuid().ToString();
-            String trackingParams = hash + "-" + Properties.Settings.Default.CacheId + "-" + Properties.Settings.Default.Started;
+            var trackingParams = $"{hash}-{Properties.Settings.Default.CacheId}-{Properties.Settings.Default.Started}";
 
-            string CurVersion = Assembly.GetExecutingAssembly().GetName().Version.ToString();
-            string CommandToSend = "/check /version " + CurVersion + " /cacheId " + trackingParams;
+            var CurVersion = Assembly.GetExecutingAssembly().GetName().Version;
+            var CommandToSend = $"/check /version {CurVersion} /cacheId {trackingParams}";
```

Comment:
this line could go under the block `if (CurVersion.Major == 0)` because it is only needed for the other cases.

Reply:
Done!